bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -1199,6 +1237,40 @@ public ByteBuf getExplicitLac(long ledgerId) throws IOException, Bookie.NoLedger
         return lac;
     }
 
+    /**
+     * Forces (sync) data on a ledger on the Journal.
```

Comment:
Nit: Consider changing this to -> " force sync given 'ledgerId' entries on the journal to the disk.

Reply:
Okay